cmd/lncli/profile.go

```diff
@@ -108,18 +116,22 @@ func getGlobalOptions(ctx *cli.Context) (*profileEntry, error) {
 func profileFromContext(ctx *cli.Context, store bool) (*profileEntry, error) {
 	// Parse the paths of the cert and macaroon. This will validate the
 	// chain and network value as well.
+	tlsCert := []byte("")
```

Comment:
nit: move this variable just above the `if` in which it is used. And don't assign a default value, just `var tlsCert []byte` is fine I think.